CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[NEW]** Add `KangarooSection`, a fullscreen without spacing on small devices section
```

Comment:
Do we need a better name :) ?

Reply:
The PR looks good technically, let's have a quick poll at next standup/weekly ?
We can move forward with this name maybe in the meantime.
My real concern is that this one time joke turns into a "I ve no name, let's use a funny name" practice.
Maybe @pith has some good naming inspiration on this one ?

Reply:
`MediaSection` ?

Reply:
We didn't wanted to put the `Mobile` name in its as it's more tied on a breakpoint than a device. `MapSection` isn't quite good too as it'll be used to the profile picture modification too. We already have a `MediaContentSection`. I don't know what to do :'(